addons/stock/static/src/js/counted_quantity_widget.js

```diff
@@ -0,0 +1,37 @@
+odoo.define('stock.counted_quantity_widget', function (require) {
+    'use strict';
+
+const BasicFields = require('web.basic_fields');
+const fieldRegistry = require('web.field_registry');
+
+const CountedQuantityWidgetField = BasicFields.FieldFloat.extend({
+    supportedFieldTypes: ['float'],
+
+     _renderReadonly: function () {
+        if (this.recordData.inventory_entered) {
+            this.$el.text(this._formatValue(this.recordData.inventory_quantity));
+        } else {
+            this.$el.text("");
+        }
+    },
+
+    _onChange: function () {
+        this._rpc({
+            model: 'stock.quant',
+            method: 'set_inventory_entered',
+            args: [this.recordData.id],
+            context: {'inventory_mode': true},
+        });
+        this._super.apply(this);
+    },
+
+    _isSameValue: function() {
+        // We want to trigger the update of the view when inserting 0
+        return false;
```

Comment:
only if the value is zero. Otherwise it will trigger compute when it's not needed